pycfmodel/constants.py

```diff
@@ -21,37 +21,6 @@
     "Ref",
 }
 
-CONDITION_MODIFIERS = {"ForAllValues", "ForAnyValue", "IfExists"}
```

Comment:
Removing this 2 constants is a major change

Reply:
Yep, I added the changelog